src/App.js

```diff
@@ -492,88 +523,110 @@ class App extends Component {
-            ) : (
-              <Routes childProps={childProps} />
             )}
-            <Onboarding
-              isOpen={this.state.isTourOpen}
-              onRequestClose={this.closeTour}
-              showCloseButton
-            />
-            <Dialog
-              style={{
-                margin: "auto",
+          >
+            <Box
+              sx={{
+                width: "100vw",
+                height: "100vh",
+                bgcolor: "background.default",
+                color: "text.primary",
+                overflow: "scroll",
               }}
-              open={this.state.loading}
-              TransitionComponent={Transition}
-              keepMounted
-              disableEscapeKeyDown
-              fullScreen
-              fullWidth
-              hideBackdrop={false}
-              aria-labelledby="loading"
-              aria-describedby="Please wait while the page loads"
             >
-              <LoadingModal />
-            </Dialog>
-          </>
-        </Sentry.ErrorBoundary>
+              {this.state.isAuthenticated ? (
+                <>
+                  <div
+                    className="sticky-logout"
+                    style={{
+                      filter: theme.palette.mode === "dark" ? "invert()" : "",
+                    }}
+                    onClick={this.handleLogout}
+                  >
+                    <GrLogout style={{ height: "20px" }} />
+                  </div>
+
+                  <div className="root-padding">
+                    <LeftNav
+                      chosenLanguage={this.state.chosenLanguage}
+                      updateState={this.setState.bind(this)}
+                      user={this.state.user}
+                      athletes={this.state.athletes}
+                    />
+
+                    <Routes childProps={childProps} />
+                  </div>
+                  <div className="sticky-nav">
+                    <div className="sticky-chat">
+                      <Image
+                        src={question}
+                        onClick={(event) => {
+                          event.preventDefault();
+                          this.setState({ isTourOpen: true });
+                        }}
+                        height="40"
+                        width="40"
+                        circle
+                        className="sticky-btn"
+                        style={{
+                          marginRight: 12,
+                          cursor: "pointer",
+                          filter: "grayscale(100%)",
+                          outline: "2px solid white",
+                          border: "2px solid transparent",
+                        }}
+                      />
+                    </div>
+                    <div id="myBottomNav" className="bottom-nav">
+                      <BottomNav user={this.state.user} />
+                    </div>
+                  </div>
+                </>
```

Comment:
Nit: Is this <> fragment needed in the ? clause of the conditional render, if there is a div wrapping around the entire element? There's no performance hit, but I suppose it's not necessary.

Reply:
Yep, it's needed to prevent eslint from complaining. 

<img width="447" alt="Screen Shot 2022-03-31 at 6 38 12 PM" src="https://user-images.githubusercontent.com/84106309/161161078-2ccd1b1c-7163-417f-ae1f-27fed3dc3e06.png">